### ISSUE TITLE
allSelectedText 

### ISSUE THREAD

**Issue by xero** [ehynds/jquery-ui-multiselect-widget#339]
## feature request:

hello,
i just started using your jquery multiSelect widget. it's awesome!
i have figured out almost everything i need, except one thing.

you have properties for "selectedText" and "noneSelectedText", but where is the "allSelectedText" option?

i was trying to hack my own looking at binding to the checkAll and uncheckAll events, but that's not good enough. what if someone selects everything, then unselects just one option?

basically i'm using it as a locations filter. when nothing is selected the noneSelectedText is: "Select Location(s)", the selectedText is: "# of # Locations Selected", but i'd also  like an "All Locations" text for if everything is selected. 

thanx a lot for this great tool!
